pkg/api/types.go

```diff
@@ -767,9 +771,9 @@ const (
 	// The default compute resource namespace for all standard resource types.
 	DefaultResourceNamespace = "kubernetes.io"
 	// CPU, in cores. (500m = .5 cores)
-	ResourceCPU ResourceName = "cpu"
+	ResourceCPU ResourceName = DefaultResourceNamespace + "/cpu"
```

Comment:
It looks like maybe we could change pkg/conversion/converter.go to do the default field copy and then override select fields based on registered conversions.